runelite-client/src/main/java/net/runelite/client/Notifier.java

```diff
@@ -68,7 +76,9 @@
 
 	// Notifier properties
 	private static final Color FLASH_COLOR = new Color(255, 0, 0, 70);
-	private static final int FLASH_DURATION = 2000;
+	private static final int MINIMUM_FLASH_DURATION_MILLIS = 2000;
+	private static final int MINIMUM_FLASH_DURATION_TICKS = MINIMUM_FLASH_DURATION_MILLIS / Constants.CLIENT_TICK_LENGTH;
+	private FlashNotification flashNotification;
```

Comment:
You should just be able to access the config directly from `processFlash` instead of storing this.

Reply:
I stored it because otherwise it would be reading it from config every frame. So I think this way is better. What do you think?

Reply:
It looks to me like it would only be reading from config each frame when it was flashing, but not otherwise.

Reply:
Yes that's right only when it's flashing, so I guess it's not that bad. There is also a slight (and this may be the epitome of slightness) advantage in that you can update the setting while it's flashing and have it take affect immediately.

I went ahead and changed it as you suggested.